pkg/storage/schema.go

```diff
@@ -112,7 +112,7 @@ func (sm *SchemaManager) migrateSchema(ctx context.Context, d migrationDirection
 	if err != nil {
 		return 0, err
 	}
-	version, err := sm.GetSchemaVersion(ctx)
+	version, _, err := sm.GetSchemaVersion(ctx)
```

Comment:
question: shall we include schema state in schema up and down api calls as well? or as part of response code? I am not sure if consumers would be able to self diagnose that schema is in bad state

Reply:
Consumers do not get new version of schema if the migration fails and schema is marked as dirty. They get non-200 response with specific error message.

Reply:
On top of that, adding the field  would increase the scope of breaking change to scripts that use the API.